spyder/app/cli_options.py

```diff
@@ -157,6 +157,13 @@ def get_options(argv=None):
         default=None,
         help="Choose a configuration directory to use for Spyder."
     )
+    parser.add_argument(
+        '--connect-to-kernel',
+        type=str,
+        dest="connection_file",
+        default=None,
+        help="Start Spyder and connect to an existing ipython kernel."
```

Comment:
```suggestion
        help=(
            "Connect to an existing kernel whose info is available in a "
            "kernel-*.json file"
        )
```

Improve help for this new option.